kyo-data/shared/src/main/scala/kyo/Tag.scala

```diff
@@ -565,8 +568,9 @@ object Tag:
 
             val concreteFlag =
                 staticDB("0") match
-                    case ClassEntry(_, _, params, _) if params.isEmpty => "*"
-                    case _                                             => "."
+                    case ClassEntry(_, _, params, _) if params.isEmpty     => "*"
+                    case OpaqueEntry(_, _, _, _, params) if params.isEmpty => "*"
```

Comment:
I realized that we can consider opaque types without type params as concrete as well